### ISSUE TITLE
saving PiB scripts

### ISSUE THREAD

**Issue by Barry4679** [davidferguson/pibakery#43]
It would good if there was enhancement to PIBakery Export & Import facility.

* a confirmation warning if attempt to attempt to close a unsaved changed project ... triggered by close of PiBakery or by Import of another script
* a Save button (enabled by an Import), which saves to original location ... it is too easy to destroy a PiB script by exporting to incorrect file

**Comment by davidferguson** [comment]
This is a good idea - I'll probably have it so the `Export` button changes to `Save` if you've imported a recipe, would that be okay?

**Comment by Barry4679** [comment]
yes, as long as you could also alter the location, so that you could clone to a test version.
